### ISSUE TITLE
Prefix parameter not passed to test-reporter if coverageLocations is empty

### ISSUE THREAD

**Issue by matthewshirley** [paambaati/codeclimate-action#265]
**Describe the bug**
I have set the test reporter to upload without specifying the coverageLocations parameter. In this scenario, the prefix is never added to the after-build call.

**Version of `codeclimate-action` you're using**
`v2.5.7`

**Example links**

#### CI Step

```
      - name: Publish coverage to CodeClimate
        uses: paambaati/codeclimate-action@v2.7.4
        env:
          CC_TEST_REPORTER_ID: demo
        with:
          debug: true
          workingDirectory: ./example_example
          prefix: /home/runner/work/example_example/example_example
```

#### Environment

```
 with:
    debug: true
    workingDirectory: ./example_example
    prefix: /home/runner/work/example_example/example_example
  env:
    ...
```

#### Debug

```
time="2020-12-04T03:52:45Z" level=debug msg="about to run format-coverage" 
time="2020-12-04T03:52:45Z" level=debug msg="searching for a formatter to use" 
time="2020-12-04T03:52:45Z" level=debug msg="checking gocov formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking search path c.out for gocov formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking jacoco formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking search path jacoco.xml for jacoco formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking lcov formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking search path coverage/lcov.info for lcov formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking clover formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking search path build/logs/clover.xml for clover formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking search path clover.xml for clover formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking cobertura formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking search path cobertura.xml for cobertura formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking coverage.py formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="checking search path coverage.xml for coverage.py formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="found file coverage.xml for coverage.py formatter" 
time="2020-12-04T03:52:45Z" level=debug msg="couldn't load committed at from ENV, trying git..." 
time="2020-12-04T03:52:45Z" level=debug msg="creating test file report for /home/runner/work/example_example/example_example/example_example/conftest.py" 
time="2020-12-04T03:52:45Z" level=info msg="trimming with prefix /home/runner/work/example_example/example_example/example_example/" 
```

**Expected behavior**
The prefix to be added to the after-build call.

**Comment by roobre** [comment]
I just came across this very same issue. Because of https://github.com/codeclimate/test-reporter/issues/378, this breaks all go tests outside of `$GOPATH`, which is the default behavior for Github Actions.

**Comment by paambaati** [comment]
@roobre The fix provided by @matthewshirley has now landed in [`v2.7.5`](https://github.com/paambaati/codeclimate-action/releases/tag/v2.7.5). Can you try that and see if that fixes your issue? If it doesn't, feel free to reopen this issue.

**Comment by roobre** [comment]
Seems to be working nice and smooth now, many thanks @paambaati @matthewshirley! ❤️
